install.sh

```diff
@@ -0,0 +1,266 @@
+#                                                                             #
+#                        Centreon Gorgone daemon module                       #
+#                                                                             #
+#                                                                             #
+###############################################################################
+
+__EOT__
+
+## displaying the license
+echo -e "\nPlease read the license.\\n\\tPress enter to continue."
+read
+tput clear
+more "$BASE_DIR/LICENSE.txt"
+
+yes_no_default "Do you accept the license ?"
+if [ "$?" -ne 0 ] ; then
+    echo_info "As you did not accept the license, we cannot continue."
+    log "INFO" "Installation aborted - License not accepted"
+    exit 1
+else
+    log "INFO" "Accepted the license"
+fi
+
+## Test all binaries
+BINARIES="rm cp mv ${CHMOD} ${CHOWN} echo more mkdir find ${GREP} ${CAT} ${SED}"
+
+line="------------------------------------------------------------------------"
+
+echo "$line"
+echo -e "\tChecking all needed binaries"
+echo "$line"
+
+binary_fail="0"
+for binary in $BINARIES; do
+	if [ ! -e ${binary} ] ; then 
+		pathfind "$binary"
+		if [ "$?" -eq 0 ] ; then
+			echo_success "${binary}" "$ok"
+		else 
+			echo_failure "${binary}" "$fail"
+			log "ERROR" "\$binary not found in \$PATH"
+			binary_fail=1
+		fi
+	else
+		echo_success "${binary}" "$ok"
+	fi
+done
+
+## Script stop if one binary wasn't found
+if [ "$binary_fail" -eq 1 ] ; then
+	echo_failure "Please check failing binary and retry"
+	echo -e "\tThe logs are available in this file :\n$LOG_FILE"
+	exit 1
+fi
+
+echo -e "\n$line"
+echo -e "\tChecking the mandatory folders"
+echo -e "$line"
+
+## check filesystem space
+check_disk_space
+
+## define destination folders
+allow_creation_of_missing_folders
+locate_gorgone_logdir
+locate_gorgone_varlib
+locate_gorgone_etcdir
+locate_gorgone_bindir
+locate_gorgone_perldir
+locate_cron_d
+locate_logrotate_d
+locate_system_d
+locate_sysconfig
+
+echo "$line"
+echo -e "\tChecking the required users"
+echo "$line"
+
+## create gorgone user
+check_gorgone_group
+check_gorgone_user
+
+#----
+## installation of Gorgone files
+#----
+echo "$line"
+echo -e "\tInstalling Gorgone daemon"
+echo "$line"
+
+## Copy the files in destination folders and modify rights
+copy_and_modify_rights "$BASE_DIR/config/systemd" "gorgoned-service" "$SYSTEM_D" "gorgoned.service" "664"
+copy_and_modify_rights "$BASE_DIR/config/systemd" "gorgoned-sysconfig" "$SYSCONFIG" "gorgoned" "664"
+copy_and_modify_rights "$BASE_DIR/config/logrotate" "gorgoned" "$LOGROTATE_D" "gorgoned" "775"
+copy_and_modify_rights "$BASE_DIR/packaging" "config.yaml" "$GORGONE_ETC" "config.yaml" "755"
+copy_and_modify_rights "$BASE_DIR" "gorgoned" "$GORGONE_BINDIR" "gorgoned" "755"
+copy_and_modify_rights "$BASE_DIR/contrib" "gorgone_config_init.pl" "/usr/local/bin" "gorgone_config_init.pl" "775"
```

Comment:
Use bin dir defined by user?
```suggestion
copy_and_modify_rights "$BASE_DIR/contrib" "gorgone_config_init.pl" "$GORGONE_BINDIR" "gorgone_config_init.pl" "775"
```